### PR TITLE
Trim dataref and command paths

### LINKED ISSUE #1424: x-plane dataref strings could trim off for possible trailing spaces
It's quite easy to copy and paste an X-plane dataref or command string and accidentally have a space or enter in the code field. 

This causes the dataref to fail.

Mobiflight should probably check for any trailing whitespace and trim the string to make it less prone to user error.

### PR DESCRIPTION
**Pull request by DocMoebiuz**
fixes #1424 

- [x] add trim to saving and loading the `.Path` value

### CODE REVIEW

**Comment by DocMoebiuz** [comment]
@tigert - this should fix your issue

File: UI/Panels/Config/HubHopPresetPanel.cs [hunk 1]
```diff
@@ -233,7 +233,7 @@ internal void syncToConfig(OutputConfigItem config)
                 config.SimConnectValue.Value = SimVarNameTextBox.Text.ToLF();
```
Comment (Koseng) [inline UI/Panels/Config/HubHopPresetPanel.cs:233]: Also add .Trim() for MSFS? Won't hurt.

Comment (DocMoebiuz) [reply UI/Panels/Config/HubHopPresetPanel.cs:233]: wasn't sure. i will check with Jaime

**Review by Koseng — COMMENTED** [review]
In general the .Trim() could also make sense on MSFS for manually entered presets.

Lines: 233, 256

**Comment by DocMoebiuz** [comment]
I didn't do the MSFS2020 change, I was worried to break something :D

**Pull request merged**